7Cav-Alive.Altis/scripts/baseCleanup/clearItemsBASE_switch.sqf

```diff
@@ -0,0 +1,13 @@
+CLIENT_BaseCleanup = {

```

Comment:
See above comments on what to do with this file (rename it and put the other functions inside it).